core/utxo/utxo.go

```diff
@@ -1846,7 +1851,7 @@ func (uv *UtxoVM) Walk(blockid []byte, ledgerPrune bool) error {
 				return updateErr
 			}
 		}
-		updateErr := uv.updateLatestBlockid(undoBlk.PreHash, batch, "error occurs when undo blocks")
+		updateErr := uv.updateLatestBlockid(undoBlk.PreHash, block.Height, batch, "error occurs when undo blocks")
```

Comment:
The height and block don't match here,will cause abnormal.

Reply:
fixed